api/capi/include/nnstreamer.h

```diff
@@ -235,12 +224,12 @@ typedef struct {
  * @detail If an application wants to accept data outputs of an NNStreamer stream, use this callback to get data from the stream. Note that the buffer may be deallocated after the return and this is synchronously called. Thus, if you need the data afterwards, copy the data to another buffer and return fast. Do not hold too much time in the callback. It is recommended to use very small tensors at sinks.
  * @since_tizen 5.5
  * @remarks The @a data can be used only in the callback. To use outside, make a copy.
- * @remarks The @a info can be used only in the callback. To use outside, make a copy.
- * @param[in] data The contents of the tensor output (a single frame. tensor/tensors). Number of tensors is determined by data->num_tensors. Note that max num_tensors is 16 (ML_TENSOR_SIZE_LIMIT).
- * @param[in] info The cardinality, dimension, and type of given tensor/tensors.
+ * @remarks The @a info_h can be used only in the callback. To use outside, make a copy.
```

Comment:
* info_h -> info

> ../nnstreamer/api/capi/include/nnstreamer.h:227: Question:
  @a is used to refer to parameters, but the 'info_h' parameter was not found in this function. Are you sure this is correct?

Reply:
It was changed to info.